acq4/devices/Camera/Camera.py

```diff
@@ -811,10 +832,16 @@ def run(self):
                             'pixelSize': [ps[0] * binning[0], ps[1] * binning[1]],  ## size of image pixel
                             'objective': ss.get('objective', None),
                             'deviceTransform': transform,
+                            'lightSource': lightSourceDescription
```

Comment:
I'm using the lightSourceStatus to indicate the on/off state of the light sources, and the "lightSource" item to give the description (type, power, wavelength, etc) of the light sources.